app/components/stepper.js

```diff
@@ -51,8 +60,8 @@ export default class StepperComponent extends Component {
   @action startHandler() {
     if (this.login.isLoggedIn && !this.login.isLoading) {
       localStorage.setItem('id', this.login.userData.id);
-      localStorage.setItem('first_name', this.login.userData.first_name);
-      localStorage.setItem('last_name', this.login.userData.last_name);
+      localStorage.setItem('first_name', this.login.userData?.first_name);
+      localStorage.setItem('last_name', this.login.userData?.last_name);
```

Comment:
Why is this optional?
What should happend when userData is not present?

Reply:
Can we do it like this ?


```
@action startHandler() {
  if (this.login.isLoggedIn && !this.login.isLoading) {
    if (this.login.userData) {
      localStorage.setItem('id', this.login.userData.id);
      localStorage.setItem('first_name', this.login.userData.first_name || 'Unknown');
      localStorage.setItem('last_name', this.login.userData.last_name || 'User');
      this.incrementStep();
    } else {
      alert('User data is not available. Please try logging in again.');
    }
  } else {
    alert('You must be logged in to continue');
  }
}

```

for more safety check

Reply:
But why?

Reply:
If user logged in then we have user data 
if (this.login.isLoggedIn && !this.login.isLoading) {